libs/SalesforceSDK/src/com/salesforce/androidsdk/rest/RestRequest.java

```diff
@@ -383,9 +395,25 @@ private static StringBuilder toCsv(List<String> fieldList) {
 	 * @throws IOException 
 	 */
 	public static RestRequest getRequestForUpdate(String apiVersion, String objectType, String objectId, Map<String, Object> fields) throws IOException  {
```

Comment:
Added support in RestRequest for specifying if-unmodified-since header during update
Initially I wanted to use it during leave-if-changed type sync up but that will have to wait until the server supports the same header for delete